extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -50,11 +45,14 @@ amp-sticky-ad {
 
 .-amp-sticky-ad-close-button {
   position: absolute;
+  display: block;
   height: 32px;
   width: 32px;
   background-color: #fff;
   color: grey;
-  font-size: 100%;
+  font-size: 150%;
```

Comment:
This works well here, but I am a little worried `font-size` goes greater than 100%.